org.fsf.pspp.yml

```diff
@@ -0,0 +1,33 @@
+app-id: org.fsf.pspp
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: psppire
+modules:
+  - name: spread-sheet-widget
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://alpha.gnu.org/gnu/ssw/spread-sheet-widget-0.8.tar.gz
+        sha256: 8589d8298fcf3b5850d0968b04801a4f40faf0555544f6cc9d954b0162e9954b
+  - name: gsl
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/gsl/gsl-2.7.1.tar.gz
+        sha256: dcb0fbd43048832b757ff9942691a8dd70026d5da0ff85601e52687f6deeb34b
+  - name: pspp
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/pspp/pspp-1.6.0.tar.gz
+        sha256: 8d5b51dba8ad812ee93288ad664d8184f164d3b394d2f9c09fcb39f86e0b0a12
+finish-args:
```

Comment:
the finish args session is usually above `modules`.

Reply:
OK, I pushed a commit to adjust the order.